src/app/entity/EntityRelation.ts

```diff
@@ -0,0 +1,7 @@
+import {Entity} from './entity';
+
+export abstract class EntityRelation extends Entity {
```

Comment:
In which way is `EntityRelation` different from `Entity`? Does it really require a separate class for this?